packages/cosmos_ui_components/lib/components/cosmos_image_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosImageButton extends StatelessWidget {
+  final String assetUrl;
+  final VoidCallback onPressed;
+
+  const CosmosImageButton({
+    Key? key,
+    required this.assetUrl,
+    required this.onPressed,
```

Comment:
could we follow other widgets here and use `onTapXX` naming? here, simply `onTap` will be perfect.
(other widgets can have more specialized naming, like `onTapWallet`, `onTapEdit` etc. rather than `onPressedWallet` `onEditPressed` etc.)

Reply:
Updated